Makefile

```diff
@@ -72,10 +125,6 @@ verify-modules: modules
 		echo "go module files are out of date, please commit the changes to go.mod and go.sum"; exit 1; \
 	fi
 
-
-changelog:
```

Comment:
Not sure why we remove this?

Reply:
The gcp and aws plugin don't have a `changelog` target in the Makefile and also the script is not present.
The csi and azure on the otherhand does have both.
All of the plugins do have a folder called `changelogs` though.

With the plugins being so similar, IMHO the Makefile and workflows should be as similar as possible?
May I suggest adding the `changelog` target and script also to the gcp and aws repo, or if they arent't used to generate the changelog remove it here and the csi plugin?

Reply:
I'd prefer to keep it as is and use the [issue](https://github.com/vmware-tanzu/velero/issues/4647) to tackle it.